### PR TITLE
Import conversational-toolkit code

### PR DESCRIPTION
**Pull request by alalamav**
- imports conversational-toolkit code
- tweaked paths in pre-commit hook to reflect new directory structure
- removed duplicate code-of-conduct.md and contributing.md and linked to toplevel docs from README
- added instructions to set up and tear down a server python virtual environment in getting-started.md
- added server python virtual environment bin and lib directories to .gitignore 
- locally tested client and server e2e

### CODE REVIEW

File: conversational-toolkit/client/README.md
Comment (adamconnors) [inline conversational-toolkit/client/README.md:1]: I guess we could remove this to avoid confusion?

File: conversational-toolkit/client/package-lock.json
Comment (adamconnors) [inline conversational-toolkit/client/package-lock.json:1]: I believe these are supposed to be committed -- just sanity checking that's right.

File: conversational-toolkit/server/.pylintrc
Comment (adamconnors) [inline conversational-toolkit/server/.pylintrc:1]: I'm not sure why this is showing up red, but I double checked and we do need it for pylint to run correctly.

**Review by adamconnors — APPROVED** [review]
Looks fabulous !

I think there's one file we might be able to drop, but dble check there might be a reason to keep it.

I ran through the readme with a clean client and everything seemed to go smoothly.

File: conversational-toolkit/client/README.md
Comment (alalamav) [reply conversational-toolkit/client/README.md:1]: Done in 9d09322.

**Pull request merged**